### PR TITLE
Feature - Issue 709 - Moves tx-module table to RDS modules table

### PR DESCRIPTION
**Pull request by richmahn**

### CODE REVIEW

File: tests/door43_tools_tests/test_templaters.py [hunk 1]
```diff
@@ -148,7 +151,9 @@ def verifyBibleTemplater(self, success, expect_success, output_folder, files_to_
             file_name = os.path.join(output_folder, file_to_verify)
             self.assertTrue(os.path.isfile(file_name), 'file not found: {0}'.format(file_name))
 
-    def verifyTaTemplater(self, success, expect_success, output_folder, filesToVerify=[]):
+    def verifyTaTemplater(self, success, expect_success, output_folder, filesToVerify=None):
+        if not filesToVerify:
+            filesToVerify = None
```
Comment (PhotoNomad0) [inline tests/door43_tools_tests/test_templaters.py]: @richmahn - this doesn't make sense.  Is it supposed to be like line 128?

**Review by PhotoNomad0 — APPROVED** [review]
LGTM

File: tests/door43_tools_tests/test_templaters.py (hunk 1 above)
Comment (richmahn) [reply tests/door43_tools_tests/test_templaters.py]: Not suppose to have mutatable structures in the definition.

Comment (richmahn) [reply tests/door43_tools_tests/test_templaters.py]: Not suppose to have mutatable structures in the definition.

**Pull request merged**